src/browser/modules/DBMSInfo/DatabaseSelector.jsx

```diff
@@ -32,18 +32,25 @@ const Select = styled.select`
   color: ${props => props.theme.inputText};
 `
 
+const EMPTY_OPTION = 'Select db to use'
+
 export const DatabaseSelector = ({
   databases = [],
-  defaultDb = 'neo4j',
   selectedDb = '',
   onChange = () => {}
 }) => {
   if (!Array.isArray(databases) || databases.length < 1) {
     return null
   }
   const selectionChange = ({ target }) => {
+    if (target.value === EMPTY_OPTION) {
+      return
+    }
     onChange(target.value)
   }
+  if (!selectedDb) {
+    databases.unshift({ name: EMPTY_OPTION, status: null })
```

Comment:
this is mutating the array, consider using an immutable alternative

Reply:
👍